distributor-api-v1/environments.md

```diff
@@ -4,13 +4,13 @@
 
 This environment is meant to be used during implementation of the client applications. We have prepared one hotel where you can fetch availability and create reservation. You can use the following information to access it:
 
-* **Platform Address** - `https://demo.mews.li`
+* **Platform Address** - `https://api.mews-demo.com`
```

Comment:
For the purposes of documentation, we could rename this to `ApiBaseUrl` to make it clearer. It will also unify the naming with `ImageBaseUrl`

Reply:
Very good point https://mews.myjetbrains.com/youtrack/issue/RND-52482 I will be writing the docs for PaymentRequest flow via Distributor next week so I will be touching the docs and will improve it in the meanwhile

Reply:
👍